src/internal/connector/graph/service_helper.go

```diff
@@ -82,6 +82,11 @@ func (handler *LoggingMiddleware) Intercept(
 		return resp, err
 	}
 
+	// special case for supportability: log all throttling cases.
+	if resp.StatusCode == http.StatusTooManyRequests {
+		logger.Ctx(ctx).Infow("graph api throttling", "method", req.Method, "url", req.URL)
```

Comment:
is it safe to log the URL? It seems like it could contain potentially sensitive info

Reply:
Something we'll have to tackle when we address log sanatiziation.